src/ensembl/production/metadata/api/models/genome.py

```diff
@@ -26,6 +26,7 @@ class Genome(Base):
     assembly_id = Column(ForeignKey("assembly.assembly_id"), nullable=False, index=True)
     organism_id = Column(ForeignKey("organism.organism_id"), nullable=False, index=True)
     created = Column(DATETIME(fsp=6), nullable=False)
+    is_best = Column(TINYINT(1), nullable=False)
```

Comment:
if not `nullable`, I believe it should have a default to `0`, right?

Reply:
That is a good point that extends beyond is_best. i have added it to all the not null booleans.